x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -123,6 +125,39 @@ export function savedObjectsRoutes(
     })
   );
 
+  /**
+   * @apiGroup JobSavedObjects
+   *
+   * @api {get} /api/ml/saved_objects/sync_needed Check whether job saved objects need synchronizing
+   * @apiName syncCheckSchema
```

Comment:
This looks wrong. Should just be `SyncCheck` I think - with initial uppercase later to match the other routes.

It also needs adding to `apidoc.json`.

Reply:
updated in https://github.com/elastic/kibana/pull/112693/commits/eb72d7b653622f3d67c114b4e92d52f94dab201b